_data/companies/brutale.yml

```
@@ -0,0 +1,6 @@
+name: Brutale Engineering SL
+wfh: Mandatory
```

```suggestion
wfh: Required
```

Switching terms for consistency